README.adoc

```diff
@@ -244,12 +276,32 @@ to retrieve the system properties for the specific localhost.
 Notice that the results from the two URLs are identical because the `inventory` service gets its results from
 calling the `system` service.
 
-[role="code_command", subs="quotes"]
+To see the application metrics, point your browser to https://localhost:9443/metrics/application[https://localhost:9443/metrics/application^]. Log in as the `admin` user with `adminpwd` as the password.
```

Comment:
Change to:
"To see the application metrics, go to the https://localhost:9443/metrics/application[https://localhost:9443/metrics/application^] URL. Log in as the `admin` user, and use`adminpwd` as the password."

Reply:
Updated